### PR TITLE
Further improvements for the search & some bug-fixes

### PR DESCRIPTION
**Pull request by Olliwehr**
* Make the search breadcrumbs more present (e.g. with different colors)
* Add detailed "No-Search-Results-Info"
* Fix mobile scroll bugs in the BrowseView
* Center Collection Title in BrowseView
* Make it possible to search for pictures based on the name of a collection the pictures are located in

### CODE REVIEW

File: projects/bp-gallery/src/components/views/search/SearchBreadcrumbs.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@
     &[search-type='description'] {
       background: $descriptionColor;
     }
-    &[search-type='decade'] {
```
Comment (BenediktV) [inline projects/bp-gallery/src/components/views/search/SearchBreadcrumbs.scss:23]: Shouldn't we maybe keep 'decade' as well? When clicking on an Item in the DecadesList, there is still a breadcrumb (with search-type: 'decade')

Comment (Olliwehr) [reply projects/bp-gallery/src/components/views/search/SearchBreadcrumbs.scss:23]: That is sth. we should keep yeah 👍

File: projects/bp-gallery/src/components/views/search/SearchBreadcrumbs.tsx [hunk 2]
```diff
@@ -21,7 +22,7 @@ const SearchBreadcrumbs = ({ searchParams }: { searchParams: URLSearchParams })
   const searchParamValues: SearchParams = [];
   const iconForType = (searchType: string) => {
     switch (searchType) {
-      case SearchType.DECADE:
+      case SearchType.TIME_RANGE:
```
Comment (Olliwehr) [inline projects/bp-gallery/src/components/views/search/SearchBreadcrumbs.tsx:25]:
```suggestion
      case SearchType.DECADE:
      case SearchType.TIME_RANGE:
```

As @BenediktV mentioned on SearchBreadcrumbs.scss file, we should keep the decade-handling and just use the same icon for the time-range search type.

**Review by Olliwehr — COMMENTED** [review]
These are changes that Werner will be impressed about!

**Review by BenediktV — APPROVED** [review]
🚀

**Pull request merged**